cmds/db-manager/main.go

```diff
@@ -1,250 +1,29 @@
-		}
-
-		// Ensure SQL session has implicit transactions disabled for CRDB versions 22.2+
-		sessionConfigurationSQL := ""
-		if crdbVersion.Compare(*semver.New("22.2.0")) >= 0 {
-			sessionConfigurationSQL = "SET enable_implicit_transaction_for_batch_statements = false;\n"
-		}
-
-		migrationSQL := sessionConfigurationSQL + fmt.Sprintf("USE %s;\n", dbName) + string(rawMigrationSQL)
-
-		// Execute migration step
-		if _, err := crdb.Pool.Exec(context.Background(), migrationSQL); err != nil {
-			log.Panicf("Failed to execute %s migration step %s: %v", dbName, fullFilePath, err)
-		}
-
-		// Update current state
-		if dbName == "defaultdb" && newVersion.String() == "4.0.0" && newCurrentStepIndex > currentStepIndex {
-			// RID database changes from `defaultdb` to `rid` when moving up to 4.0.0
-			dbName = "rid"
-		}
-		if dbName == "rid" && currentVersion.String() == "4.0.0" && newCurrentStepIndex < currentStepIndex {
-			// RID database changes from `rid` to `defaultdb` when moving down from 4.0.0
-			dbName = "defaultdb"
-		}
-		actualVersion, err := crdb.GetVersion(context.Background(), dbName)
-		if err != nil {
-			log.Panicf("Failed to get current database version for %s: %v", dbName, err)
-		}
-		if !actualVersion.Equal(*newVersion) {
-			log.Panicf("Migration %s should have migrated %s schema version %v to %v, but instead resulted in %v", fullFilePath, dbName, currentVersion, newVersion, currentVersion)
-		}
-		currentVersion = actualVersion
-		currentStepIndex = newCurrentStepIndex
-	}
-
-	log.Printf("Final %s version: %v", dbName, currentVersion)
-}
-
-func enumerateMigrationSteps(path *string) ([]MigrationStep, error) {
-	steps := make(map[semver.Version]MigrationStep)
-
-	// Identify files defining version migration steps
-	files, err := os.ReadDir(*path)
-	if err != nil {
-		return make([]MigrationStep, 0), stacktrace.Propagate(err, "Failed to read schema files directory")
-	}
-	r := regexp.MustCompile(migrationStepRegexp)
-	for _, file := range files {
-		if !file.IsDir() {
-			match := r.FindStringSubmatch(file.Name())
-			if len(match) > 0 {
-				v := *semver.New(match[2])
-				step := steps[v]
-				step.version = v
-				if match[1] == "upto" {
-					step.upToFile = file.Name()
-				} else if match[1] == "downfrom" {
-					step.downFromFile = file.Name()
-				} else {
-					return make([]MigrationStep, 0), fmt.Errorf("Unexpected migration step prefix: %s", match[1])
-				}
-				steps[v] = step
-			}
-		}
-	}
-
-	// Sort versions in ascending order
-	versions := make([]*semver.Version, len(steps))
-	i := 0
-	for k := range steps {
-		v := steps[k].version
-		versions[i] = &v
-		i++
-	}
-	semver.Sort(versions)
-
-	// Render sorted step list
-	result := make([]MigrationStep, len(versions)+1)
-	result[0].version = *semver.New("0.0.0")
-	for i := 0; i < len(versions); i++ {
-		result[i+1] = steps[*versions[i]]
+	DBManagerCmd = &cobra.Command{
+		Use:   "db-manager",
+		Short: "DSS database management",
```

Comment:
```suggestion
		Short: "DSS database management utility",
```

(nit: I think this is a better answer to "what is db-manager?")